framework/CodeInterfaces/Generic/GenericParser.py

```diff
@@ -52,7 +52,8 @@ def __init__(self,inputFiles,prefix='$RAVEN-',postfix='$',defaultDelim=':', form
       infileName = inputFile.getFilename()#os.path.basename(inputFile)
       self.segments[infileName] = []
       if not os.path.exists(inputFile.getAbsFile()):
-        raise IOError('Input file not found: '+inputFile)
+        ## Make sure to cast the inputFile to a string as it may be File object.
+        raise IOError('Input file not found: ' + str(inputFile))
```

Comment:
Okay, but still good to know for the messageHandler. I probably still would have used the long-form syntax. 👍